plugins/org.wso2.developerstudio.eclipse.artifact.connector/plugin.xml

```diff
@@ -86,9 +86,9 @@
                 </enablement>
             </commonWizard>
     </extension>        
-   <extension point="org.wso2.developerstudio.eclipse.dashboad.link">	
+   <!--extension point="org.wso2.developerstudio.eclipse.dashboad.link">	
```

Comment:
If this in not going to be used in the feature, better to remove it rather than commenting it